ai/instill/v0/README.mdx

```diff
@@ -310,10 +310,3 @@ Generate image from input text prompt and image.
 | Output | ID | Type | Description |
 | :--- | :--- | :--- | :--- |
 | Images | `images` | array[string] | Images |
-
```

Comment:
Please fetch the latest `compogen`. These newlines are produced by changes in the `compogen` template to improve its readability (we don't care about readability in `README.mdx`, only on its HTML version, which removes redundant spaces.

Reply:
@jvallesm 
Ah, I forgot to sync with you yesterday.
I did fetch the latest compogen.
But, it seems conflict with pre-commit hook now.

<img width="989" alt="image" src="https://github.com/instill-ai/component/assets/64747455/d88f88c2-16bb-42d6-bbed-39090f1328d2">

Reply:
@jvallesm 
I did fetch the latest compogen.
But, it seems conflict with pre-commit hook now.
When I commit, those redundant lines are delete by pre-commit hook.

Reply:
Which one should we follow now?

Reply:
@chuang8511 @jvallesm 
I add a rules to exclude the pre-commit-hooks for mdx files.